pastis/matrix_generation/matrix_from_efields.py

```diff
@@ -367,9 +397,9 @@ def _simulator_matrix_single_mode(which_dm, number_all_modes, wfe_aber, simulato
         hcipy.imshow_field(opd_map, grid=simulator.aperture.grid, mask=simulator.aperture, cmap='RdBu')
         plt.savefig(os.path.join(resDir, 'OTE_images', opd_name + '.pdf'))
 
-    return efield_focal_plane.electric_field
-
+    return efield_focal_plane.electric_field, efield_wfs_plane
```

Comment:
This change is not trivial, as it would also require the function `_rst_matrix_single_mode()` to return two objects rather than just one. Let's talk about how we can solve this.

In any case, if we chose to keep this change, the E-field in the WFS plane would very likely also need to return the actual efield attribute `.electric_field`, like it is done for the focal-plane E-field.